part-i-getting-started/quick-start.md

```diff
@@ -0,0 +1,355 @@
+
+### Commands and Events
+As mentioned, we need a command to issue the Giftcard. This command will contain the identifier of the Giftcard and the initial amount. Let's build one:
+
+```java
+public class IssueCmd {
+
+    private final String id;
+    private final Integer amount;
+    
+    public IssueCmd(String id, Integer amount) {
+        this.id = id;
+        this.amount = amount;
+    }
+    
+    public String getId() {
+        return id;
+    }
+    
+    public Integer getAmount() {
+        return amount;
+    }
+}
+```
+
+We'll name an event that will be triggered after this command is successfuly processed `IssuedEvt`:
+
+```java
+public class IssuedEvt {
+
+    private final String id;
+    private final Integer amount;
+    
+    public IssuedEvt(String id, Integer amount) {
+        this.id = id;
+        this.amount = amount;
+    }
+    
+    public String getId() {
+        return id;
+    }
+    
+    public Integer getAmount() {
+        return amount;
+    }
+}
+```
+
+Similarly, we'll create `RedeemCmd` (do note that `amount` is amount to be deducted from the Giftcard) and `RedeemEvt`:
+
+```java
+public class RedeemCmd {
+
+    @TargetAggregateIdentifier // (1)
+    private final String id;
+    private final Integer amount;
+    
+    public RedeemCmd(String id, Integer amount) {
+        this.id = id;
+        this.amount = amount;
+    }
+    
+    public String getId() {
+        return id;
+    }
+        
+    public Integer getAmount() {
+        return amount;
+    }
+}
+```
+
+(1) `@TargetAggregateIdentifier` annotation is used by Axon to find the correct Giftcard aggregate instance.
+
+```java
+public class RedeemedEvt {
+    
+    private final String id;
+    private final Integer amount;
+    
+    public RedeemedEvt(String id, Integer amount) {
+        this.id = id;
+        this.amount = amount;
+    }
+    
+    public String getId() {
+        return id;
+    }
+        
+    public Integer getAmount() {
+        return amount;
+    }
+}
+```
+
+> **Note** Command/Event definition looks a bit cumbersome, this can be avoided using hierarchy between commands/events, or using libraries like [lombok](https://projectlombok.org/), or using [kotlin](https://kotlinlang.org/) which provides more concise way of defining commands/events.
+
+### Aggregate
+
+Now that we have our commands and events which represent the API of our application, we can create Giftcard aggregate:
+
+```java
+public class GiftCard {
+
+    @AggregateIdentifier // (1)
+    private String id;
+    private int remainingValue;
+
+    public GiftCard() {
+        // (2)
+    }
+	
+	@CommandHandler
+    public GiftCard(IssueCmd cmd) {        
+        if(cmd.getAmount() <= 0) throw new IllegalArgumentException("amount <= 0");
+        AggregateLifecycle.apply(new IssuedEvt(cmd.getId(), cmd.getAmount()));
+    }
+	
+	@EventSourcingHandler
```

Comment:
Similarly, why not add a `// (4)`  comment to describe the `@EventSourcingHandler` later on?